api/package.json

```diff
@@ -23,7 +23,8 @@
         "reindex": "ts-node prisma/reindex.ts",
         "format:check": "npx prettier --check src/",
         "format:write": "npx prettier --write src/",
-        "lint": "eslint src/"
+        "lint:check": "eslint src/",
```

Comment:
we should probably run this in github actions now?

Reply:
hmm the checks dont make sense if force running prettier and linting (as the linting 'fixes' prettier), either we assume  prettier and linting have run (due to pre-commit hook) or we can only run one of them (lint check perhaps better than nothing? then if someone has managed to push unprettiered/unlinted code this will fail?)

Reply:
done